portfolio/src/main/webapp/appdev.html

```diff
@@ -29,7 +29,7 @@ <h1>App Development</h1>
           The final application I developed for iOS was an infinite runner game called Number Runner Infinity.
       </p>
       <img src="images/NR.png" width="50%" height="50%"/>
-      <p>You can click <a href="index.html">here</a> to return to the main page.</p>
+      <p>You can click <a href="index.html" class=hlink>here</a> to return to the main page.</p>
```

Comment:
missing double quotes?

Reply:
Not sure if this a html best practices thing or not, but the class declaration works with the double quotes, I can change it however as it appears to work both ways.

Reply:
*with or without the double quotes

Reply:
I made this change on my latest push.